intelmq/lib/bot.py

```diff
@@ -394,5 +399,56 @@ def __load_harmonization_configuration(self):
                             "Key %s is not valid." % _key)
 
 
+class ParserBot(Bot):
+
+    def parse_csv(self, report):
+        """
+        CSV parser.
+        """
+        raw_report = utils.base64_decode(report.get("raw"))
+        for line in csv.reader(io.StringIO(raw_report)):
```

Comment:
Not implemented and used yet. No converted parsers had a csv format with headers.